treeherder/model/fixtures/repository.json

```diff
@@ -1310,7 +1310,7 @@
         "dvcs_type": "git",
         "name": "servo-try-taskcluster",
         "url": "https://github.com/servo/servo",
-        "branch": "try-taskcluster",
+        "branch": "(disabled, use servo-try instead)",
```

Comment:
Should we mark this as ``active_status: "onhold"`` now?  Or do you still need it for a while longer?  Otherwise, perhaps a follow-up PR to deactivate it?

Reply:
I'll do a follow-up PR to deactivate it.

Reply:
I lied, I added a new commit.

Reply:
Awesome.  I'll merge as soon as Travis finishes.  Thanks for the PR!

Reply:
Oh right, I didn’t think of changing the status instead. With that, we don’t need to change the branch, right?